src/common/utils/MetaKeyUtils.cpp

```diff
@@ -1152,9 +1152,30 @@ GraphSpaceID MetaKeyUtils::parseLocalIdSpace(folly::StringPiece rawData) {
   return *reinterpret_cast<const GraphSpaceID*>(rawData.data() + offset);
 }
 
-GraphSpaceID MetaKeyUtils::parseDiskPartsSpace(folly::StringPiece rawData) {
+HostAddr MetaKeyUtils::parseDiskPartsHost(const folly::StringPiece& rawData) {
```

Comment:
LGTM. Since the change contains a lot of string manipulation, can you make comments about example input data and how the input is parsed? So people can understand how the return value is generated without reading the code in the future. Thank you!